lxd/instance/drivers/driver_lxc.go

```diff
@@ -2232,7 +2232,7 @@ func (d *lxc) startCommon() (string, []func() error, error) {
 	}
 
 	if snapName != "" && expiry != nil {
-		err := d.snapshot(snapName, expiry)
+		err := d.snapshot(snapName, expiry, "")
```

Comment:
@kadinsayani we should pass in the root volume mode here for explicitness i think.